ProcessLib/PhaseField/PhaseFieldFEM-impl.h

```diff
@@ -22,52 +22,46 @@ template <typename ShapeFunction, typename IntegrationMethod,
 void PhaseFieldLocalAssembler<ShapeFunction, IntegrationMethod,
                               DisplacementDim>::
     assembleWithJacobianForStaggeredScheme(
-        double const t, double const dt, std::vector<double> const& local_xdot,
-        const double dxdot_dx, const double dx_dx, int const process_id,
+        double const t, double const dt, Eigen::VectorXd const& local_x,
+        std::vector<double> const& local_xdot, const double dxdot_dx,
+        const double dx_dx, int const process_id,
         std::vector<double>& local_M_data, std::vector<double>& local_K_data,
         std::vector<double>& local_b_data, std::vector<double>& local_Jac_data,
-        LocalCoupledSolutions const& local_coupled_solutions)
+        LocalCoupledSolutions const& /*local_coupled_solutions*/)
 {
     // For the equations with phase field.
     if (process_id == 1)
     {
         assembleWithJacobianPhaseFiledEquations(
```

Comment:
I know that this is not touched by the PR, but shouldn't that be
```suggestion
        assembleWithJacobianPhaseFieldEquations(
```
?

Reply:
:eagle: :eye: :trophy:

Reply:
I didn't even see the formatting is broken. My point was the name of the function (Filed vs. Field).

Reply:
Ah, OK. I'll push the change directly to master. :+1: